examples/anyons.jl

```diff
@@ -6,18 +6,19 @@ using DFTK
 using StaticArrays
 using Plots
 
-## Unit cell. Having one of the lattice vectors as zero means a 2D system
+# Unit cell. Having one of the lattice vectors as zero means a 2D system
```

Comment:
Meh, these are technical comments that don't really merit a separate markdown section

Reply:
More worryingly, the plot in https://docs.dftk.org/dev/examples/anyons/ doesn't show up anymore, @mfherbst do you know what's up with that?